src/composite.comp

```diff
@@ -22,13 +22,13 @@ uniform layers_t {
     vec2 u_scale[MaxLayers];
     vec2 u_offset[MaxLayers];
     float u_opacity[MaxLayers];
-    float u_borderAlpha[MaxLayers];
+    uint u_borderMask;
     uint u_frameId;
 };
 
 layout(binding = 1) uniform sampler2D s_samplers[MaxLayers];
 
-layout(binding = 5) uniform sampler2D s_ycbcr_samplers[MaxLayers];
+layout(binding = 1 + MaxLayers) uniform sampler2D s_ycbcr_samplers[MaxLayers];
```

Comment:
Please do this instead: https://github.com/DadSchoorse/gamescope/commit/7dedf6ce675ca98f2bca4a4c4ff138a7357350cd

Reply:
Thanks, TIL that arrays only count for 1 binding. Crazy!